BUILD-LINUX.md

```diff
@@ -127,7 +127,7 @@ cmake --build . --config Release
 ## 4. Start Sonic Pi
 
 Finally, you can run your newly compiled `Sonic Pi` app within the `build`
-directly either by double clicking it in the Finder or via the terminal
+directly either by double clicking it in your file manager or via the terminal
 (from within the `build` directory):
```

Comment:
Hi, I wanted to point out that when I built on ubuntu, the compiled sonic pi app was in `build/gui/qt/sonic-pi`, not `build/sonic-pi`.

Reply:
Hi, I wanted to make a PR about this. I built Sonic Pi successfully on Pop!_OS 20.04 and the executable is located in the `build/gui/qt/sonic-pi` directory.

I think it should be mentioned in the documentation because I had to use the `find` command and thought there was a build error!  ;)

Reply:
Yes, i've added that to the MR with https://github.com/sonic-pi-net/sonic-pi/commit/c2ff495d6d35f9ae01bdad2076d9368a6e7acb0c.

Just wait for review.

Reply:
Thanks, just saw it now! :+1: